www/grpc/proto/network.proto

```diff
@@ -71,10 +61,14 @@ message GetNodeInfoResponse {
   repeated string local_addrs = 8;
   // List of protocols supported by the node.
   repeated string protocols = 9;
+  // Type of node pruned or full.
+  bool is_pruned = 10;
```

Comment:
To be backward compatible please don't change the field numbers.

Reply:
We didn't changed fields. we just used some unused numbers.

Reply:
There was no field for 10 and 11 before this.